### DISCUSSION TITLE
Aggregate fields

### DISCUSSION THREAD

**Discussion by craxal** [community/community#6254] [Projects and Issues]
With custom fields in projects, that brings up the question of doing some analysis based on those fields. Say, for example, I have a Cost numeric field that tracks the estimated number of days it would take to resolve a particular issue. I only have so much "capacity" for a particular milestone. It would be nice for a way to somehow aggregate the cost of all issues so that it's easy to see if my milestone has the right amount of work.

**Comment by nkolev92** [comment]
Fully agree, the same thing applies to epics as well. 
The practicality of the feature goes down significantly if you there's no means to do proper estimates.
